bin/make_err

```diff
@@ -213,18 +213,98 @@ sub create_public ($) {
     close HEADER;
 }
 
+##############################################################################
+# Create the generated portion of the H5E major message definition code
+#
+sub create_majdef ($) {
+    my $prefix = shift;         # Get the prefix for the generated file
+    my $file = "H5Emajdef.h";   # Name of file to generate
+    my $name;                   # Name of error message
+
+    # Rename previous file
+#    rename "${prefix}${file}", "${prefix}${file}~" or die "unable to make backup";
+
+    # Open new header file
+    open HEADER, ">${prefix}${file}" or die "unable to modify source";
+
+    # Create file contents
+
+    print_copyright(*HEADER);
+    print_warning(*HEADER);
+    print_startprotect(*HEADER, $file);
+
+    # Iterate over all the major errors
+    print HEADER "\n/***********************************/\n";
+    print HEADER   "/* Major error message definitions */\n";
+    print HEADER   "/***********************************/\n\n";
+    print HEADER   "/* clang-format off */\n";
+    foreach $name (sort keys %major) {
+        printf HEADER "static const H5E_msg_t ${name}_msg_s = {false, \"${major{$name}}\", H5E_MAJOR, &H5E_err_cls_s};\n";
+    }
+    print HEADER   "/* clang-format on */\n";
+
+    print_endprotect(*HEADER, $file);
+
+    # Close header file
+    close HEADER;
+}
+
+##############################################################################
+# Create the generated portion of the H5E minor message definition code
+#
+sub create_mindef ($) {
```

Comment:
This creates a static const message for each library-internal minor error message, so they aren't malloc'ed/free'd at init/term.